app/views/organization_wizzard/new_account.html.slim

```diff
@@ -0,0 +1,29 @@
+.page-header
+  h2 New organization bank account
+
+.row.previous_buttons class="#{flash[:open_form] ? 'hidden' : ''}"
+  .col-sm-6
+    p
+      ' Bank account will be created by default (with default currency)
+      br
+      br
+      = button_to 'Create default Bank account', default_account_organization_path(@organization.id), class: 'btn btn-primary', method: :get
```

Comment:
@dissident `method: :post`. Use `POST` or `PUT` or `DELETE` when you make some action and `GET` when you just show/retrive data